main/unbound/unbound.initd

```diff
@@ -10,10 +7,13 @@ description="unbound is a Domain Name Server (DNS) that is used to resolve host
 description_configtest="Run syntax tests for configuration files only."
 description_reload="Kills all children and reloads the configuration."
 
-
 UNBOUND_BINARY=${UNBOUND_BINARY:-/usr/sbin/unbound}
 UNBOUND_CHECKCONF=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}
-UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${SVCNAME}.conf}
+UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${RC_SVCNAME}.conf}
+
+command=${UNBOUND_BINARY}
+command_args=${UNBOUND_OPTS}
+command_background=yes
```

Comment:
isn't forcing  `foreground` required only when using `supervise-daemon` ?

`unbound` starts & stops just fine as is